pkg/command/service/measure_test.go

```diff
@@ -100,6 +100,40 @@ func TestNewServiceMeasureCommand(t *testing.T) {
 		assert.NilError(t, err)
 	})
 
+	t.Run("measure service with output flag", func(t *testing.T) {
+		ns := &corev1.Namespace{
```

Comment:
You should add a test for a bad output location. For instance a directory that does not exist or is not writable and deal with it with a correct message to the user

Reply:
I have refined code and add some verification, so the test has also been updated.